lib/components/gradient_container.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+
+class GradientContainer extends StatelessWidget {
+  const GradientContainer(
+      {Key? key,
+      required this.child,
+      this.accentColor = const Color(0x4411a1c1),
+      this.center = Alignment.topRight})
+      : super(key: key);
+
+  final Widget child;
+  final Color accentColor;
+  final AlignmentGeometry center;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+          gradient: RadialGradient(
+            radius: 3.5,
+            colors: [
+              Color(0xFF111130),
+              Color(0xFF111111),
```

Comment:
Maybe let's stick to the convension where we keep constants in other file. We don't have to but it definetely would make the code cleaner

Reply:
True, thanks for reminding.